src/ci/resources/papi_application.inc.conf.ctmpl

```diff
@@ -63,7 +63,7 @@ services {
   }
 }
 
-filesystems.drs.global.config.martha.url = "https://us-central1-broad-dsde-dev.cloudfunctions.net/martha_v3"
+filesystems.drs.global.config.resolver.url = "https://drshub.dsde-dev.broadinstitute.org/api/v4/drs/resolve"
```

Comment:
Can tell us about your current plan for rolling out the actual change to point everything at Dr Shub, or maybe point me to a doc? It looks like this changes our CI from running against Martha to running against The Hub, which is obviously a good thing to do on this PR, but maybe not what we want for all our PR's in the near future. 

It would also be good to confirm that this changeset still works against Martha, since that's what we'll be using in prod initially... or will we?

Reply:
In case it makes a difference, @tlangs should know that as of very recently Cromwell `develop` goes to prod every Monday as opposed to scheduled releases once every few months.

Reply:
Right now, everything deployed is still pointing to Martha. We expect non-prod envs to start pointing to DRSHub soon, but not at this time. If we don't feel like Cromwell should be talking to DRSHub for PRs yet, I can change this back.

Reply:
Yes, let's keep this pointing at Martha for now and remember to update it later when prod starts using DRSHub.